specs/cm/change-mgt-vocab.ttl

```diff
@@ -28,10 +28,15 @@
 oslc_cm:
   a owl:Ontology ;
   dcterms:title "The OSLC Change Management(CM) Vocabulary" ;
+  rdfs:label "The OSLC Change Management(CM) Vocabulary" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Change Management (CM) namespace."^^rdf:XMLLiteral ;
-  dcterms:source <https://tools.oasis-open.org/version-control/svn/oslc-ccm/trunk/specs/change-mgt/change-mgt-vocab.ttl> ;
   vann:preferredNamespacePrefix "oslc_cm" ;
-  rdfs:label "Change Management(CM)" .
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
```

Comment:
Should be commented out until publication.

Reply:
done